api/gulpfile.js

```diff
@@ -176,6 +187,3 @@ gulp.task('undeploy', gulp.series(
   'delete_cloudwatch_logs'
 ));
 
-gulp.task('bootstrap', gulp.series('deploy','generate_sample_users', 'generate_sample_data'));
```

Comment:
I would keep this bootstrap task, and expand it to have your 2-3 new sample group and user creation tasks in its sequence.